libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/GutenbergEditorFragment.java

```diff
@@ -112,11 +120,21 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
                     public void onUploadMediaButtonClicked() {
                         mEditorFragmentListener.onAddPhotoClicked();
                     }
+
                     @Override
                     public void onCapturePhotoButtonClicked() {
                         checkAndRequestCameraAndStoragePermissions();
                     }
                 },
+                new OnReattachQueryListener() {
+                    @Override
+                    public void onQueryCurrentProgressForUploadingMedia() {
+                        for (String mediaId : mUploadingMediaProgressMax.keySet()) {
```

Comment:
Hey @mzorz, what about moving this into a separate method (e.g. `updateMediaProgress` or similar) 
and also maybe we can put this `mUploadingMediaProgressMax.get(mediaId)` into variable (e.g. `int currentProgress`) before passing it to method ?

Reply:
agreed, addressed moving into a new method in 6cb4c90456 - I think the intermediate local variable is unnecessary as it's only going to be used in 1 place; we can use that when it's referenced in more than 1 place so the `.get()` method doesn't get called on every reference. Does it make sense?